docs/en/03_configuration.md

```diff
@@ -242,34 +254,41 @@ E.g.
 Or if you want to use local time in a summary field (where server time is UTC)
 
 ```php
- private static $summary_fields = [
-    'Title'                           => 'Title',
-    'Created'                         => 'Created (utc)',
-    'Created.getLocalTime.LocalValue' => 'Created (local)',
-];
+namespace App\Models;
+
+use SilverStripe\ORM\DataObject;
+
+class MyDataObject extends DataObject
+{
+    private static $summary_fields = [
+        'Title' => 'Title',
+        'Created' => 'Created (utc)',
+        'Created.getLocalTime.LocalValue' => 'Created (local)',
+    ];
+}
 ```
 
-## SilverStripe Fluent and search
+## Silverstripe `Fluent` and search
 
-### SilverStripe Core search
+### Silverstripe core search
 
-To use SilverStripe Core search with Fluent, add the `FluentSearchForm` to your configuration.
+To use Silverstripe Core search with Fluent, add the [`FluentSearchForm`](api:TractorCow\Fluent\Search\FluentSearchForm) to your configuration.
```

Comment:
This probably needs to be updated (I don't even know what a "Silverstripe Core search" is) - but that's out of scope for this PR so leave alone for now.